src/LabLog/Migrations/20170928082432_InitialCreate.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace LabLog.Migrations
+{
+    public partial class InitialCreate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "LabEvents",
+                columns: table => new
+                {
+                    Version = table.Column<int>(type: "INTEGER", nullable: false)
```

Comment:
I'm not sure this should be autoincremented, we want full control over the version to make sure two events aren't recorded in paralell. This is called optimistic locking

Reply:
This file was autogenerated based off the model? Are you supposed to edit the generated file, or modify the model so the file is generated properly?

Reply:
Automatically generated?

Reply:
Is it appropriate to edit these autogenerated files, rather than modifying the model so that this file is generated according to needs?

Reply:
All this auto generated stuff is based on the model and can be configured. Might need to have a poke at this later